activmusic/apps/uploadmgr/templates/uploadmgr/index.html

```diff
@@ -10,20 +10,24 @@
 
     {% for media in medias %}
     {% if forloop.first %}
-    <table class="table">
-        <tr>
-            <th>{% trans "Name" %}</th>
-            <th>{% trans "URL" %}</th>
-            <th class="table-col-delete">{% trans "Delete" %}</th>
-        </tr>
-    {% endif %}
-        <tr>
-            <td>{{ media.name|default:"[not downloaded yet]" }}</td>
-            <td><a href="{{ media.url }}" target="_blank">{{ media.url }}</a></td>
-            <td><input type="checkbox"/></td>
-        </tr>
-    {% if forloop.last %}
-    </table>
+    <form method="post">
+        {% csrf_token %}
+        <button class="btn btn-danger">Remove all selected tracks</button>
```

Comment:
Maybe red is too visible too. Try something more discrete. "Delete" is not an action you want your user to do.